deployment/helm/skaha/start-jupyterlab/init-jupyterlab.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# One argument expected: the skaha session ID, which makes up part of the connect url
+
```

Comment:
As this script is executed by the initcontainer which runs the user-defined image, just pointing out a security consideration that /bin/bash, cp etc could be malicious executables containing arbitrary code. 
Of course users can run their own images anyway which could contain anything (and must be fully unprivileged), so the only consideration in scope here is the potential impact of any such malicious code taking part in preparation of the passwd and group files.